### ISSUE TITLE
Adressen Export zu LibreOffice Zeichensatz falsch eingestellt

### ISSUE THREAD

**Issue by librelegal** [jlawyerorg/j-lawyer-org#703]
In der Funktion Adressen --> suchen --> Liste nach LibreOffice lauert ein einfacher, aber folgenschwerer Fehler. Bisher sind nur die Adressen aus dem angebotenen Import enthalten (z.B. Amtsgericht Dresden, Roßbachstraße 6). 

Klicke ich nun auf Exportieren nach LibreOffice wird als Zeichensatz "Westeuropäisch ISO 8859-1" vorausgewählt. Dann aber sind Satzzeichen wie ss-zett (hier irgendwie nicht darstellbar) falsch angegeben. Auch Umlaute wie in Görlitz werden falsch angezeigt.

Richtig ist Zeichensatz System, zumindest bei mir

System: Linux debian buster

Finde hier auch nichts dazu, gibt es noch eine aktuellere Anleitung?
https://www.j-lawyer.org/downloads/documentation/j-lawyer.org-UserGuide-de.html

**Comment by j-dimension** [comment]
Die Voreinstellung beim Import in LibreOffice kann ich nicht "von außen" setzen. Man könnte das also maximal dokumentieren, eine Codeänderung sehe ich hier erstmal nicht.

Behält LO denn zumindest die zuletzt gewählte Einstellung beim nächsten Import?

**Comment by librelegal** [comment]
Nein, nach Schließen von j-lawyer sowie LibreOffice (Version: 6.1.5.2
Build-ID: 1:6.1.5-3+deb10u5, CPU-Threads: 4; BS: Linux 4.19; UI-Render: Standard; VCL: gtk3_kde5; Gebietsschema: de-DE (de_DE.UTF-8); Calc: group threaded) wird erneut der  fehlerhafte Zeichensatz "Westeuropäisch ISO 8859-1"  angezeigt

**Comment by j-dimension** [comment]
Dann lasse ich das Ticket offen, kann aber nur das Verhalten dokumentieren / Hinweise für die richtigen Einstellungen geben.
